dev/storage.h

```diff
@@ -852,11 +852,11 @@ namespace sqlite_orm {
             }
 
             template<class O, class... Cols>
-            int insert(const O& o, columns_t<Cols...> cols) {
-                static_assert(cols.count > 0, "Use insert or replace with 1 argument instead");
+            int64 insert(const O& o, columns_t<Cols...> cols) {
+                static_assert(cols.count > 0, "Specify at least one column");
                 this->assert_mapped_type<O>();
                 auto statement = this->prepare(sqlite_orm::insert(std::ref(o), std::move(cols)));
-                return int(this->execute(statement));
+                return this->execute(statement);
```

Comment:
I understand that this is legacy but it doesn't mean that all my arguments have no meaning